ttnn/cpp/ttnn/operations/moreh/moreh_layer_norm_backward/device/kernels/moreh_layernorm_backward_gamma_beta_grad_kernel.cpp

```diff
@@ -0,0 +1,332 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ttnn/cpp/ttnn/deprecated/tt_dnn/kernels/compute/moreh_common.hpp"
+
+
+namespace NAMESPACE {
+void MAIN {
+    constexpr uint32_t num_cols_per_core = get_compile_time_arg_val(0);
+    constexpr uint32_t origin_H = get_compile_time_arg_val(1);
+    constexpr uint32_t origin_W = get_compile_time_arg_val(2);
+    constexpr uint32_t NCHt = get_compile_time_arg_val(3);
+    constexpr uint32_t Wt = get_compile_time_arg_val(4);
+    constexpr bool gamma_grad_has_value = get_compile_time_arg_val(5) == 1;
+    constexpr bool beta_grad_has_value = get_compile_time_arg_val(6) == 1;
+    constexpr bool is_lastdim_layernorm = get_compile_time_arg_val(7) == 1;
+    constexpr bool is_groupnorm = get_compile_time_arg_val(8) == 1;
+
+    constexpr auto cb_dy = tt::CB::c_in0;      // output_grad(==dy)
+    constexpr auto cb_x = tt::CB::c_in1;       // input(==x)
+    constexpr auto cb_mean = tt::CB::c_in2;    // mean
+    constexpr auto cb_rstd = tt::CB::c_in3;    // rstd
+    constexpr auto cb_scaler = tt::CB::c_in4;  // scaler
+    constexpr auto cb_mask_h = tt::CB::c_in5;  // mask_h
+    constexpr auto cb_mask_w = tt::CB::c_in6;  // mask_w
+
+    // Sum[y * dy]
+    constexpr auto cb_dgamma = tt::CB::c_out0;  // gamma_grad(==dgamma)
+    // Sum[dy]
+    constexpr auto cb_dbeta = tt::CB::c_out1;  // beta_grad(==dbeta)
+
+    // y = (x - mean) / rstd
```

Comment:
Could you change it like this? It's my mistake.

```suggestion
    // y = (x - mean) * rstd
```

Reply:
resolved